docs/shared/node-tutorial/5-docker-target.md

```diff
@@ -0,0 +1,127 @@
+# Node Standalone Tutorial - Part 5: Docker Target
+
+## Generate a Micro-service with a Docker File
+
+Let's create a new `orders-api` application with a docker file included:
+
+```{% command="npx nx g @nrwl/node:app orders-api --docker" path="~/products-api" %}
+>  NX  Generating @nrwl/node:application
+
+✔ Which framework do you want to use? · express
+CREATE orders-api/src/assets/.gitkeep
+CREATE orders-api/src/main.ts
+CREATE orders-api/tsconfig.app.json
+CREATE orders-api/tsconfig.json
+CREATE orders-api/project.json
+CREATE orders-api/.eslintrc.json
+CREATE orders-api/jest.config.ts
+CREATE orders-api/tsconfig.spec.json
+CREATE orders-api-e2e/project.json
+CREATE orders-api-e2e/jest.config.ts
+CREATE orders-api-e2e/src/orders-api/orders-api.spec.ts
+CREATE orders-api-e2e/src/support/global-setup.ts
+CREATE orders-api-e2e/src/support/global-teardown.ts
+CREATE orders-api-e2e/src/support/test-setup.ts
+CREATE orders-api-e2e/tsconfig.json
+CREATE orders-api-e2e/tsconfig.spec.json
+CREATE orders-api-e2e/.eslintrc.json
+CREATE orders-api/Dockerfile
+```
+
+This generator has created a basic Dockerfile for the new microservice.
+
+```{% filename="orders-api/Dockerfile" %}
+# This file is generated by Nx.
```

Comment:
Note, this might change based on our discussion here: https://app.asana.com/0/1203590581335352/1203932464064892